app/routes/v0/__init__.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint, escape, request, jsonify, Response, abort, current_app
+from datetime import datetime
+import dateutil.parser as dateparser
+from uuid import UUID
+from typing import Union, Optional
+import os
+from db import get_cases, insert_random_cases
+
+cases = Blueprint("v0.cases", __name__, url_prefix="/v0/cases")
+
+
+@cases.route("", methods=["GET"])
+def index() -> Response:
+    lat: Union[Optional[float], int] = request.args.get("lat", type=float)
+    lon: Union[Optional[float], int] = request.args.get("lon", type=float)
+    uuid: Optional[UUID] = request.args.get("uuid", type=UUID)
+    if uuid is None:
```

Comment:
For testing this would be fine. But at the end we should probably have a better validation
(for instance, the ID does not exist)

Reply:
UUIDs are not required to exist by design. They could be randomly generated on the client side. Also it is possible that the UUID has been deleted by the server. https://confluence.ito-app.org/display/IN/questions/66164

Reply:
Adding to that, this piece of code was not touched by the PR, it was just moved.